docs/proposals/default-service-plans.md

```diff
@@ -0,0 +1,544 @@
+# Default Service Plans in Service Catalog
+
+* [Abstract](#abstract)
+* [Summary](#summary)
+* [Motivation](#motivation)
+* [Goals and Non-Goals](#goals-and-non-goals)
+* [Storage Classes vs. Service Classes](#storage-classes-vs-service-classes)
+* [User Experience](#user-experience)
+  * [Cluster Operator](#cluster-operator)
+     * [Broker Installation](#broker-installation)
+     * [Cluster Customization](#cluster-customization)
+  * [Developer](#developer)
+     * [Service Class Discovery](#service-class-discovery)
+     * [Legacy Provisioning](#legacy-provisioning)
+     * [Provisioning with Defaults](#provisioning-with-defaults)
+     * [Provisioning by Service Type](#provisioning-by-service-type)
+     * [Service Instance Management](#service-instance-management)
+  * [Helm Charts](#helm-charts)
+     * [Broker](#broker)
+     * [Author](#author)
+     * [Consumer](#consumer)
+* [Implementation](#implementation)
+  * [Service Class](#service-class)
+  * [Service Plan](#service-plan)
+  * [Service Instance](#service-instance)
+  * [Service Binding](#service-binding)
+
+## Abstract
+
+Let's wholesale rip off how default storage classes for volumes work and apply
+it to Service Catalog! 😎 This allows us to specify a dependency on a type of
+service without requiring upfront knowledge of the underlying broker.
+
+> "My application requires a mysql database."
+
+> "I just want to try out this application using the recommended defaults."
+
+> "I need to encourage everyone to use the cheapest plans possible in our
+> integration environment by making a small database the default."
+
+## Summary
+
+1.  Add default provisioning and binding parameters to Service Classes.
+1.  Add a "Service Type" field to classes and plans, e.g. mysql or redis.
```

Comment:
this would have to happen at the OSB level to make it inner-operable. or use a special formatting and use tags https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#service-object

Reply:
That's something that my teammates will be discussing at the OSB F2F next week. 👍  

I'll add a section to the proposal outlining changes that I am proposing to the OSB API for v3, and ideas for interim solutions for OSB API v2.